Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -4821,6 +4821,7 @@
 		B8DBDB4E130B0F8A00F5CDB1 /* SpellingCorrectionCommand.h in Headers */ = {isa = PBXBuildFile; fileRef = B8DBDB4A130B0F8A00F5CDB1 /* SpellingCorrectionCommand.h */; };
 		B9CA3B402CAB1D3E00B2607C /* SpatialImageControls.h in Headers */ = {isa = PBXBuildFile; fileRef = B9CA3B3E2CAB1D3E00B2607C /* SpatialImageControls.h */; };
 		B9CA3B412CAB1D3E00B2607C /* SpatialImageControls.cpp in Sources */ = {isa = PBXBuildFile; fileRef = B9CA3B3F2CAB1D3E00B2607C /* SpatialImageControls.cpp */; };
+		BA79DEF12DDDF89D000DB04C /* IntegrityPolicy.h in Headers */ = {isa = PBXBuildFile; fileRef = BA79DEEF2DDDF89D000DB04C /* IntegrityPolicy.h */; settings = {ATTRIBUTES = (Private, ); }; };
```

Comment:
Now that this isn't #included in DocumentLoader.h, this can also be removed as a private header (and similarly removed from Headers.cmake).